spec/requests/api/v1/users/login_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+require "rspec/json_expectations"
+
+describe 'Login User', type: :request do
+  subject { post '/api/v1/auth/sign_in', params:, as: :json }
```

Comment:
```suggestion
  subject { post api_v1_users_sign_in_path, params:, as: :json }
```
Check if your route match with that, but the idea is to use rails paths